recirq/qcqmc/experiment.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2024 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Tuple, Union
+
+import attrs
+import cirq
+import numpy as np
+import qsimcirq
+from pytz import timezone
+
+from recirq.qcqmc import blueprint, config, data, for_refactor
+
+
+def _to_tuple(x: Optional[Iterable[float]]) -> Optional[Sequence[float]]:
```

Comment:
I would consolidate this with other tuple functions or remove it.  This is simple enough we could add it inline, 
such as `tuple(x) if x is None else x`

Reply:
Done.